### DISCUSSION TITLE
Different computers, different themes

### DISCUSSION THREAD

**Discussion by Lancing34** [SylEleuth/gruvbox-plus-icon-pack#57] [General]
I'm fairly new to Linux. I have a desktop and a laptop configured quite similarly (ubuntu). When I installed this icon theme on both, I got the beige icons on my desktop and the colored icons on my laptop. How can I fix this? I'd like them to match.

**Comment by SylEleuth** [comment]
Can You provide screenshots and explain in more details what You want exactly?

**Reply by Lancing34** [reply]
Hey! Thanks for the quick reply. I've attached screenshots. One is a picture of my panel on my laptop with the colored icons, and the other is a picture from my desktop computer with the subdued icons. I would like to be able to figure out what I need to change in order to have these both be the same theme. Such as both computers have subdued icons. I've installed them both the same way and verified all the settings match to the best of my ability.

![DesktopTheme](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/170684497/73e56e20-d8c0-47f3-ab3b-e064c2528c4c)
![LaptopTheme](https://github.com/SylEleuth/gruvbox-plus-icon-pack/assets/170684497/ba8f69fb-c640-479b-a6f2-641668ee4ce6)

**Comment by SylEleuth** [comment]
What's the screen resolution of desktop and laptop? If desktop has higher resolution can You lower it to the same as laptop and see if there is any difference?

**Reply by Lancing34** [reply]
They're both set to 1920x1080.

**Reply by SylEleuth** [reply]
Well, it's weird then. I have to test it myself first. Can You tell me what kind of panel is that so I can reproduce it?

**Reply by Lancing34** [reply]
Dash to Dock
https://extensions.gnome.org/extension/307/dash-to-dock/

Thanks for your help! I'm just as stumped as you might be.

**Comment by SylEleuth** [comment]
Are You sure that the settings are exactly the same? What I can see that if You set icons size to 24 everything looks fine, but setting it below 24 then monochrome icons will be used. If you don't want to see monochrome icons whatsoever and use only colourful ones (for apps) You would have to tinker a little bit.
In icon pack's folder find folder named `apps`. You will see several different subfolders but only 3 of theme contain icons, the rest are symbolic links linking to those folders. It decides which icons are used depending of the size. `24` folder links to `scalable`, but `22` links to `16`. If you want to use icons e.g. 22 size and still see colourful icons You would have to change its symlink to `scalable`. If You want to link all sizes to `scalable` You would also have to backup `16` folder (or delete it if You want) and make symlink folder named `16` linked to `scalable` like the others.
It's just a temporary solution that will work until next release. And then you will have to change it again

**Reply by Lancing34** [reply]
That's it! Thank you! I didn't notice that I have my icons larger on the laptop than the ones on my desktop due to the different screen sizes. Sorry for leading you down a rabbit hole, but thanks for the help!

**Reply by SylEleuth** [reply]
No worries. I installed Ubuntu and discovered a bunch of problems that I will be fixing now (trash icons is one example). Take care.
